.github/workflows/validate-snapshots.yml

```diff
@@ -0,0 +1,67 @@
+name: Validate snapshots
+on:
+  workflow_dispatch:
+  schedule: 
+    - cron: '30 9 * * 1-5'
+env:
+  PAT_TOKEN: ${{ secrets.PAT_TOKEN }}
```

Comment:
Not a biggy, but weird name as it would stand for personal access token token? It could also be a little bit more specific.

Perhaps `GITHUB_TOKEN` would be better?

Reply:
Yep good thinking, that does the trick 😉. I'll get in touch with Cam tomorrow and update the repository secret with a regenerated token